src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.5.ts

```diff
@@ -0,0 +1,121 @@
+
+              {
+                  "ConfigurationRecorders": [
+                      {
+                      "recordingGroup": {
+                          "allSupported": true,
+                          "resourceTypes": [],
+                          "includeGlobalResourceTypes": true
+                      },
+                      "roleARN": "arn:aws:iam::<AWS_Account_ID>:role/service-role/<config-role-name>",
+                      "name": "default"
+                      }
+                  ]
+              }
+
+  3. Run this command to show the status for all AWS Config recorders:
+
+          aws configservice describe-configuration-recorder-status
+
+  4. In the output, find recorders with name key matching the recorders that met criteria in step 2. Ensure that at least one of them includes "recording": true and "lastStatus": "SUCCESS"`,
+
+  rationale:
+    'The AWS configuration item history captured by AWS Config enables security analysis, resource change tracking, and compliance auditing.',
+
+  remediation: `To implement AWS Config configuration:
+  Via AWS Management Console:
+
+  1. Select the region you want to focus on in the top right of the console
+  2. Click Services
+  3. Click Config
+  4. Define which resources you want to record in the selected region
+  5. Choose to include global resources (IAM resources)
+  6. Specify an S3 bucket in the same account or in another managed AWS account
+  7. Create an SNS Topic from the same AWS account or another managed AWS account
+
+  Via AWS Command Line Interface:
+
+  1. Ensure there is an appropriate S3 bucket, SNS topic, and IAM role per the [AWS Config Service prerequisites](https://docs.aws.amazon.com/config/latest/developerguide/gs-cli-prereq.html).
+  2. Run this command to set up the configuration recorder
+
+          aws configservice subscribe --s3-bucket my-config-bucket --sns-topic arn:aws:sns:us-east-1:012345678912:my-config-notice --iam-role arn:aws:iam::012345678912:role/myConfigRole
+
+  3. Run this command to start the configuration recorder:
+
+          start-configuration-recorder --configuration-recorder-name <value>`,
+
+  references: [
+    'CCE-78917-2',
+    'CIS CSC v6.0 #1.1, #1.3, #1.4, #5.2, #11.1 - #11.3, #14.6',
+    'http://docs.aws.amazon.com/cli/latest/reference/configservice/describe-configuration-recorder-status.html',
+  ],
+  gql: `{
+    queryawsAccount { 
+      id
+      __typename
+      configurationRecorders {
+        recordingGroup {
+          allSupported
+          includeGlobalResourceTypes
+        }
+        status {
+          recording
+          lastStatus
+        }
+      }
+    }
+  }`,
+  resource: 'queryawsAccount[*]',
+  severity: 'medium',
+  conditions: {
+    path: '@.configurationRecorders',
+    array_any: {
```

Comment:
does this check its enabled for every region or do we need to do something similar that was done for guardDutyDetectors?

Reply:
looks to be correct from the rationale description.